### DISCUSSION TITLE
My GitHub pages is loading incredibly slowly

### DISCUSSION THREAD

**Discussion by sethmachine** [community/community#28274] [Other Feature Feedback, Questions, & Ideas]
Hi,

I run a personal blog hosted by GitHub pages: www.sethmachine.io.  Lately the page load time has been incredibly slow.  Whenever I try to load the page in a computer browser or my phone (off wifi), it takes 5-10 seconds for the blog to load.  

This was not the usual behavior, in the past it loaded quite fast.  I don't have a lot of images as my website is mostly text and HTML.  

How can I debug why the GitHub page for my website is loading so slow?  

Many thanks,

sethmachine

**Comment by sethmachine** [comment]
These are my pages settings for my blog:

<img width="927" alt="Screen Shot 2022-08-03 at 9 27 20 PM" src="https://user-images.githubusercontent.com/8753890/182743379-da2f7542-4d76-4e78-9106-470993b309dc.png">

**Comment by jfullstackdev** [comment]
you may want to take a look at the errors at Developer Tools

![image](https://user-images.githubusercontent.com/47092464/182744338-14bc31d6-b2de-437f-9d91-1d1582f8bcef.png)

**Reply by sethmachine** [reply]
Oof thanks for pointing this out.  I haven't changed the Javascript code ever, so I'm wondering what could have caused such a regression in execution/load time.  As when I originally published the blog page load times were fine.  It's only in the last 3 months it got incredibly slow.  

The load time seems to be at least 4-5 seconds which doesn't match the values show in the dev tools?  61ms and 72ms.  

Do you have any pointers for where I should start looking to debug this?  I'm not a Javascript/web programmer but I am familiar with debugging in languages like Java and Python.

**Reply by jfullstackdev** [reply]
about forced reflow

https://stackoverflow.com/questions/41218507/violation-long-running-javascript-task-took-xx-ms

about setTimeout handler

https://stackoverflow.com/questions/59063979/violation-settimeout-handler-took-nms

**Comment by ershantanusinha** [comment]
i am going through same with my website [Taxi in Varanasi](https://bigcab.in)

**Reply by hugolpz** [reply]
@ershantanusinha  Something is off this week. Maybe has been a long time but it takes >30 minutes and counting when it was taking one minute just few months ago.

**Comment by chshydnews** [comment]
My GitHub Pages site is loading very slowly, so I’m checking image size, scripts, cache, and mobile speed. My site, [CHS HYD News](https://chshyd.in/), shares clear U.S.-focused updates on auto, finance, tech, climate, entertainment, and gaming, so fast loading is important for a better reader experience.
